resources/protos/commit.proto

```diff
@@ -0,0 +1,22 @@
+syntax = "proto2";
+
+package tutorial;
+
+option java_package = "app";
+option java_outer_classname = "CommitProtos";
+
+message Flag {
+    string type;
+    string value;
+}
+
+message Commit {
+    optional int32 id;
```

Comment:
Does this protobuf compile at all? It should not because you must specify tags for all fields.